vignettes/migration_from_the_old_crmPack.Rmd

```diff
@@ -12,8 +12,281 @@ vignette: >
 
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
+library(tibble)
+library(kableExtra)
+library(checkmate)
 ```
 
+The following Vignette describes the changes that were introduced to the `crmPack`
+package as a result of the package's refactoring. One row in below tables
+represents a single-type, consistent change.
+
+# Class and Methods Name Changes
+
+## Classes
+
+```{r class_change_helpers, echo=FALSE}
+
+# A simple helper function that generates change class record.
+ch_class <- function(name = "", new_name = "", comment = "", add_constructor_info = TRUE) {
+  assert_string(name)
+  assert_string(new_name)
+  assert_string(comment)
+  assert_flag(add_constructor_info)
+
+  was_removed <- ifelse(nchar(new_name) == 0, TRUE, FALSE)
+  if (add_constructor_info && !was_removed) {
+    constructor_info <- "\\+ constructor corresponding update"
```

Comment:
this is not really important here.... I would just omit that super small details on constructor or not changed

Reply:
Sure, I will just remove it.